meilisearch-http/src/index_controller/index_actor/actor.rs

```diff
@@ -342,13 +354,16 @@ impl<S: IndexStore + Sync + Send> IndexActor<S> {
             .await?
             .ok_or(IndexError::UnexistingIndex)?;
 
+        let processing = self.processing.read().await;
+        let is_indexing = processing.0 && processing.1 == Some(uuid);
```

Comment:
so this becomes:
```rust
let is_indexing = processing == Some(uuid);

Reply:
I think it's rather
```rust
let is_indexing = *processing == Some(uuid);
```

Your option doesn't compile 🙂 
```
error[E0369]: binary operation `==` cannot be applied to type `tokio::sync::RwLockReadGuard<'_, std::option::Option<Uuid>>`
   --> meilisearch-http/src/index_controller/index_actor/actor.rs:358:38
    |
358 |         let is_indexing = processing == Some(uuid);
    |                           ---------- ^^ ---------- std::option::Option<Uuid>
    |                           |
    |                           tokio::sync::RwLockReadGuard<'_, std::option::Option<Uuid>>
```